source/renderer/app/components/staking/delegation-setup-wizard/DelegationStepsChooseWalletDialog.tsx

```diff
@@ -125,25 +150,13 @@ export default class DelegationStepsChooseWalletDialog extends Component<
       minDelegationFunds,
       onClose,
       onBack,
-      isWalletAcceptable,
       numberOfStakePools,
       getStakePoolById,
     } = this.props;
     const selectedWallet: Wallet | null | undefined = wallets.find(
       (wallet: Wallet) => wallet && wallet.id === selectedWalletId
     );
-    const { amount, reward, isRestoring } = selectedWallet || {};
-    let errorMessage;
-
-    if (selectedWallet && !isWalletAcceptable(amount, reward)) {
-      // Wallet is restoring
-      if (isRestoring) errorMessage = messages.errorRestoringWallet;
```

Comment:
This is where the bug was. We were only assigning the error message that the wallet is restoring, when wallet didn't meet the criteria to be selected.